main.js

```diff
@@ -1,7 +1,8 @@
 (function(){
   var node = document.querySelector("div");
   var app = Elm.Main.embed(node, {
-      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : ""
+      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : "",
```

Comment:
Though I can see this wasn't added in this PR so we can address it in the next PR to be made to elm-program-builder